components/raftstore/src/store/txn_ext.rs

```diff
@@ -117,6 +123,10 @@ pub struct PeerPessimisticLocks {
     pub version: u64,
     /// Estimated memory used by the pessimistic locks.
     pub memory_size: usize,
+    /// Shrinking configuration
```

Comment:
There could be compatible risks introducing new configurations in elder LTS versions.